index.html

```diff
@@ -14,54 +14,61 @@
     <title>Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="header-desktop">
+      <nav>
       <h2 class="welcome"><a href="#">Welcome</a></h2>
-      <nav><i class="fa fa-bars" aria-hidden="true"></i></nav>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <ul class="desktop-nav">
+          <li><a href="#portfolio-home">Portfolio</a></li>
+          <li><a href="#about-section">About</a></li>
+          <li><a href="#contact-section">Contact</a></li>
+        </ul>
+      </nav>
+      <section class="home-page">
```

Comment:
- [ ] The mobile background image of this section is correct but please notice that the background image should be aligned to the edge of the page so there shouldn't be a space between the image and the edge. currently you have some margin on the `header` that is causing the background image on this `section` tag to not line up to the edge
## Your Design 
![image](https://user-images.githubusercontent.com/18730106/144668010-0274b343-0cd9-4749-8bc1-c49e985937e9.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144667926-50e38736-6721-45cf-841c-f9fd49995505.png)